share/ext/libunwind/jni/Application.mk

```diff
@@ -33,6 +33,6 @@
 #NDK_TOOLCHAIN_VERSION := clang
 #APP_STL := c++_static
 
-APP_ABI := arm64-v8a
+APP_ABI := arm64-v8a #armeabi-v7a
 APP_CXXFLAGS := -std=c++11 -frtti -D__STDC_LIMIT_MACROS -D_THREAD_SAFE
-APP_PLATFORM := android-9
+APP_PLATFORM := android-19
```

Comment:
no, but I would like to keep it for now, because it will be useful for `libunwind**stack**` building
see `Libunwind`
in https://docs.google.com/document/d/1uguu07VuUvBRQvJ4T_NnrfNl7q8JHZD1LF7IuePkhy4/edit#heading=h.mfeghlxitqk0